sdk/cosmosdb/arm-cosmosdb/CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # Release History
-
-## 16.0.1 (Unreleased)
-
+    
+## 16.1.0 (2024-09-14)
+    
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-
+  - Added Interface ErrorAdditionalInfo
+  - Added Interface ErrorDetail
+  - Added Interface ErrorResponseAutoGenerated
+  - Interface RestoreParametersBase has a new optional parameter restoreWithTtlDisabled
+  - Enum KnownServerVersion has a new value Seven0
```

Comment:
what does the swagger look like for this?

Reply:
added a new value 7.0 in their latest stable swagger: https://github.com/kazrael2119/azure-rest-api-specs/blob/572a19175617f6be83d90bfc79dd77f2eb9e355d/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2024-08-15/cosmos-db.json#L9626-L9627

Reply:
I agreed Server0 is not perfect but I prefer to keep it as it is. If we need to refine the enum key name it's better to rename all keys which would be a breaking for us, and I think it is not worth a breaking with an Knonw enum interface change. Maybe we could pay attention for further other breakings and re-consider the name again.

https://github.com/Azure/azure-sdk-for-js/blob/a9cb49d04efd0745dd67ead826698b9f0c29b6a2/sdk/cosmosdb/arm-cosmosdb/src/models/index.ts#L4009-L4025